qa/tasks/mon_thrash.py

```diff
@@ -307,6 +313,27 @@ def do_thrash(self):
                     delay=self.thrash_delay))
                 time.sleep(self.thrash_delay)
 
+        #status after thrashing
+        status = self.mds_cluster.status()
+
+        # make sure everyone is in active, standby, or standby-replay
+        log.info('Wait for all MDSs to reach steady state...')
```

Comment:
I don't see a reason to wait for all MDSs to reach a stable state? We just need to check if any MDSs have failed.